dev_tools/utils/nxdl_utils.py

```diff
@@ -127,23 +127,21 @@ def get_nx_namefit(hdf_name, name, name_any=False):
     """
     if name == hdf_name:
         return len(name) * 2
+    if re.search(r"^[0-9]", hdf_name) is not None:
```

Comment:
@sanbrock I'm not sure whether it is the right approach to guard clause this here. Generally, it should not be allowed in the file in either the definition _or_ the hdf file and here we would just not match even though the damage is already done somewhere else. We could also leave it out here and would also match incorrect hdf paths with numbers in the beginning.

Reply:
This is also used for validating nxdl.xml files, so I think we could leave it here.